aiohttp/http_writer.py

```diff
@@ -157,7 +157,9 @@ async def drain(self) -> None:
 def _py_serialize_headers(status_line: str,
                           headers: 'CIMultiDict[str]') -> bytes:
     line = status_line + '\r\n' + ''.join(
-        [k + ': ' + v + '\r\n' for k, v in headers.items()])
+        [k + ': ' + (str(v) if type(v) is int else v) + '\r\n'  # no subclasses like bool
```

Comment:
It's probably cleaner to have the conditional conversion added to a helper function.